data/src/main/java/org/oppia/android/data/backends/gae/model/GaeFeedbackReport.kt

```diff
@@ -0,0 +1,20 @@
+package org.oppia.android.data.backends.gae.model
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * Data class for the feedback report sent by the Android app to remote storage.
+ * TODO(#2801): Link backend domain model
+ */
+@JsonClass(generateAdapter = true)
+data class GaeFeedbackReport constructor(
+
+  @Json(name = "api_key") val apiKey: String,
+  @Json(name = "report_creation_timestamp_ms") val reportCreationTimestampMs: Double?,
```

Comment:
Why not a ``Long?``?

Reply:
Ah, I think I had originally changed this to a double in the proto but Long does make more sense